CHANGELOG.md

```diff
@@ -14,11 +14,13 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Added
 
 - [Doc] Add explanation for the limitation of the `useDevicePermissionStatus` hook in storybook.
+- Add support to override event reporter for client event ingestion once enabled in the Amazon Chime SDK for JavaScript. For more information check the [Client Event Ingestion guide](https://aws.github.io/amazon-chime-sdk-js/modules/clientevent_ingestion.htmlhttps://aws.github.io/amazon-chime-sdk-js/modules/clientevent_ingestion.html) in the Amazon Chime SDK for JavaScript.
```

Comment:
Probably a document generator issue? In a new revision, all the "Client Event Ingestion guide" links point to the wrong link:

```
https://aws.github.io/amazon-chime-sdk-js/modules/clientevent_ingestion.htmlhttps://aws.github.io/amazon-chime-sdk-js/modules/clientevent_ingestion.html
```

Reply:
That is weird. Let me check. Sorry about that but I had fixed this in this revision.

Reply:
Checked and looks fixed now. Not sure, how that happened.